docs/pages/aggregator.mdx

```diff
@@ -2,159 +2,72 @@
 
 This is what `ERC7786Aggregator` brings to the table. The logic is similar to a multisig, where a threshold of signers must sign a transaction in order to execute it, in the aggregator the message sent from the source chain must be sucessfully sent through different gateways before being executed on the destination chain.
 
 ## How it works
 
-The Aggregator supports both the `IERC7786GatewaySource` and `IERC7786Receiver` interfaces, i.e., someone sending a message using the aggregator interacts with it in the same way they would interact with an adapter or a native bridge that uses the standard.
+The aggregator contract implements both the `IERC7786GatewaySource` and `IERC7786Receiver` interfaces. This means that from a caller’s perspective, sending a message through the aggregator works the same way as if they were interacting with any single ERC7786-compliant bridge or adapter. The difference is that instead of forwarding the message to just one endpoint, the aggregator dispatches it across multiple gateways.
+In the diagram below it's possible to see how a 2-out-of-3 aggregator works:
+
+![Aggregator diagram](https://raw.githubusercontent.com/luiz-lvj/erc-7786/technical/docs/assets/aggregator-diagram.jpg)
+
+### Configuring the Aggregator
+
+When the aggregator contract is first deployed, it accepts a list of initial gateways along with a threshold value. These gateways must be addresses of contracts that support the `IERC7786GatewaySource` interface. The threshold, on the other hand, is used to define the number of confirmations needed before a message is considered valid. During deployment, the constructor adds each gateway to an internal collection and sets the threshold to the provided value.
+
+After initialization, the contract owner retains the ability to adjust these settings:
+
+ - **Add or Remove Gateways**: The owner can introduce a new gateway address into the aggregator’s registry or remove an existing one. This flexibility allows the aggregator to evolve over time, accommodating new bridging solutions or removing deprecated ones.
+
+ - **Update Threshold**: The owner can update the threshold whenever the system’s needs change. Depending on your protocol’s requirements, you might want a higher threshold for added security (e.g., requiring multiple gateways to confirm a message) or a lower threshold for quicker approvals and reduced costs.
+
+Because these functions can alter the fundamental behavior of how messages are dispatched and confirmed, they are restricted to the contract owner. By placing these controls in the hands of a trusted administrator, the aggregator can adapt to new conditions without compromising security.
```

Comment:
"Trusted administrator" is not a positive thing, it's neutral at best, so we should be careful with the wording here. I'd replace with:

```suggestion
Because these functions can alter the fundamental behavior of how messages are dispatched and confirmed, they are restricted to the contract owner. Note that the security of the aggregator relies on an honest owner, so carefully consider what are acceptable trust assumptions in the context of your protocol.
```